contribute/developer-guide.md

```diff
@@ -27,11 +27,14 @@ npm install -g yarn
 
 ## Download Grafana
 
-We recommend using Go to download the source code for the Grafana project:
+We recommend using the Git command-line interface to download the source code for the Grafana project:
 
-1. Add `export GOPATH=$HOME/go/` to the bottom of your `$HOME/.bash_profile`.
-1. Open a terminal and run `go get github.com/grafana/grafana` in your terminal. This command downloads, and installs Grafana to your `$GOPATH`.
-1. Open `$GOPATH/src/github.com/grafana/grafana` in your favorite code editor.
+1. Open a terminal and run `git clone https://github.com/grafana/grafana.git`. This command downloads Grafana to a new `grafana` directory in your current directory.
+1. Open the `grafana` directory in your favorite code editor.
+
+For alternative ways of cloning the Grafana repository, please refer to [GitHub's cloning a repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) documentation.
+
+We do not recommend using Go's `go get` to download Grafana. Recent versions of Go have added behavior which isn't compatible with the way the Grafana repository is structured.
```

Comment:
```suggestion
We don't recommend using Go's `go get` to download Grafana. Recent versions of Go have added behavior which isn't compatible with the way the Grafana repository is structured.
```

Reply:
I'm not a fan of contracting the emphasized part of a sentence. I now took a peek at our [documentation style guide](https://github.com/grafana/grafana/blob/34c2f440dbd701e31faa0949ddfc8921a8c6f460/contribute/style-guides/documentation-style-guide.md) and there isn't anything about this in particular, but we're using both don't and do not within that file.

I'll keep it as I wrote it for now, but I'm open to changing it given a rationale 🙂

Reply:
I also prefer "do not" here, agree with @sakjur.

Reply:
I would also change this to a warning.

> **Warning:** Do not use `go get` to download Grafana. Recent versions...

Reply:
```suggestion
**Warning:** Do not use `go get` to download Grafana. Recent versions of Go have added behavior which isn't compatible with the way the Grafana repository is structured.
```